### ISSUE TITLE
detect imported logger instances

### ISSUE THREAD

**Issue by jkowalleck** [adamchainz/flake8-logging#84]
### Description

for some reason, issues on imported `Logger` instances are not properly found.

file `some_library/__init__.py`:
```py
__all__ = ['LOGGER']
import logging
LOGGER = logging.getLogger(__name__)
LOGGER.addHandler(NullHandler())
# expected a LOG012  << thrown as expected ✔️
LOGGER.debug('foo %s', 1, 2)
```

file `some_library/foo.py`:
```py
from . import LOGGER
# expected a LOG012  << **NOT** thrown as expected ❌
LOGGER.debug('foo %s', 1, 2)
```

**Comment by adamchainz** [comment]
Loggers defined in another file are not detected as logger instances. This is a limitation of Flake8's single-file-at-a-time approach.

Perhaps we could infer a restricted set of names as logger instances, such as `LOGGER` or `LOG`. But I’m cautious against creating false positives.

If you’d like to try create a PR, it could be helpful to see what changes that would require.

**Comment by jkowalleck** [comment]
for now, this workaround helped:
```py
from typing import TYPE_CHECKING

if TYPE_CHECKING:
  from logging import getLogger
  LOGGER = getLogger(__name__)
else:
  from . import LOGGER

# expected a LOG012  << thrown as expected ✔️
LOGGER.log('foo %s', 1, 2)
  
```

**Comment by jkowalleck** [comment]
>  This is a limitation of Flake8's single-file-at-a-time approach. 

then maybe feature to annotation would help. like 

```
from . import MyLoggerInstance, something_else  # Logger:MyLoggerInstance
```

**Comment by adamchainz** [comment]
I think that would be quite tiresome and error prone.

Can I ask why you don’t use the pattern of one logger per module?

**Comment by jkowalleck** [comment]
> Can I ask why you don’t use the pattern of one logger per module?

A logger might be injectable - and therefore is to be shared.

**Comment by adamchainz** [comment]
Closing because it’s “synthetic”.
